### ISSUE TITLE
Document decision to use Ubuntu for PDS

### ISSUE THREAD

**Issue by Jeeppler** [Daimler/sechub#930]
Document the decision of using Ubuntu for all PDS solutions base images. For more information see: https://github.com/Daimler/sechub/pull/922#pullrequestreview-841298574

**Comment by sven-dmlr** [comment]
Meanwhile, Debian is the base image for PDS solutions.

@Jeeppler Can we close this issue?

**Comment by Jeeppler** [comment]
@sven-dmlr no, this issue cannot be closed. We should document our decision somewhere.
